spot-ml/src/main/scala/org/apache/spot/proxy/ProxySuspiciousConnectsAnalysis.scala

```diff
@@ -40,66 +41,45 @@ object ProxySuspiciousConnectsAnalysis {
     * @param logger       Logs execution progress, information and errors for user.
     */
   def run(config: SuspiciousConnectsConfig, sparkContext: SparkContext, sqlContext: SQLContext, logger: Logger,
-          inputProxyRecords: DataFrame) = {
+          inputProxyRecords: DataFrame): SuspiciousConnectsAnalysisResults = {
 
     logger.info("Starting proxy suspicious connects analysis.")
 
-    val cleanProxyRecords = filterAndSelectCleanProxyRecords(inputProxyRecords)
+    val proxyRecords = filterRecords(inputProxyRecords)
+      .select(InSchema: _*)
+      .na.fill(DefaultUserAgent, Seq(UserAgent))
+      .na.fill(DefaultResponseContentType, Seq(ResponseContentType))
```

Comment:
I don't think so, it has been there for a while but no documentation. I'm going to add a JIRA issue for that documentation.